ui/src/components/LogIn/index.js

```diff
@@ -11,8 +11,17 @@ export default class LogIn extends Component {
     super(props);
     document.title = "Log In - PLynx";
     this.state = {
+      email: '',
       username: '',
-      password: ''
+      password: '',
+      confpassword: '',
+      login: true,
```

Comment:
Can you please use constants instead? I didn't understand what it meant for a while